padertorch/train/hooks.py

```diff
@@ -61,6 +60,23 @@ class Hook:
     def priority(self):
         return Priority.DEFAULT
 
+    @property
+    def uid(self):
+        """
+        A unique ID of a hook.
+
+        The default `uid` disallows to register a state full hook more than one
+        in the trainer. Hooks that may registered more than once should
+        overwrite this property (e.g. `ModelAttributeAnnealingHook`)
+        """
```

Comment:
What do you think about such a docstring?

Reply:
I like. Thanks.